src/csharp/Pester/Test.cs

```diff
@@ -36,6 +36,9 @@ public Test()
         public string ExpandedPath { get; set; }
 
         public string Result { get; set; }
+        public string ExpectedValue { get; set; }
+        public string ActualValue { get; set; }
+        public string BecauseValue { get; set; }
```

Comment:
Could you provide an example of where this would be used? `-Because` values are likely specifically written to match the assertion error template, so unsure if it's useful by itself.

Reply:
It's mostly to pass through so that plug-ins like the vscode extension can surface them without having to parse the message.

Reply:
Github didn't highlight my selection too well, so just clarifying that this was specific to `BecauseValue`. :) Will the extension use that at all?

The other comment was about all three.

Reply:
@fflaten I planned to have an option to add the "because" to the test result comparison to make it more surfaced, and it was very little effort to surface it in addition to the others so I went ahead and included it.

Reply:
I plan to reformat it to its own object per @fflaten's feedback so all the context will be preserved.

Reply:
Fixed 83d0352